### DISCUSSION TITLE
How to refer/cite the clay work

### DISCUSSION THREAD

**Discussion by CarlosGomes98** [Clay-foundation/model#323] [General]
Hi! I would be interested in pointing to some of the work the Clay team has been doing in a paper. Specifically, [this](https://explore.madewithclay.org/) demo. I couldn't find an example of a preference for how to cite this. Would you have one?

Thanks!

**Comment by adamjstewart** [comment]
Since the work was never published, I usually just cite the documentation:
```bibtex
@unpublished{clay,
    author={{Clay Foundation}},
    title={Clay foundation model},
    note={https://clay-foundation.github.io/model/},
    year={2023},
}
```
Could also cite a blog post I suppose. Also worth specifying which model version you use unless you are citing the model in general.
